build.ps1

```diff
@@ -46,14 +46,19 @@ if($Clean.IsPresent) {
     Pop-Location
 }
 
-# Build step
+# Common step required by both build and test
 Find-Dotnet
+Restore-NuGetPackage
```

Comment:
Why do we need this? When you do a `dotnet publish` on the sln file in the root, it will `restore` all of the containing projects.